### PR TITLE
First pass at asking about academic qualifications

### PR DESCRIPTION
**Pull request by edwardhorsford**
Conditionally shows a page asking about academic qualifications when recommending a trainee for QTS or EYTS.

The page is only shown for routes where you might also get an academic qualification - so assessment only routes are excluded.

This works for Postgraduate courses only initially.

New question:
Screenshot 2022-01-07 at 14 11 42

Summary page:
Screenshot 2022-01-07 at 14 12 09
ad-8dd9-73cac053d4f2.png">

Record:
Screenshot 2022-01-07 at 14 12 58

A record for a route which doesn't have academic outcomes:
Screenshot 2022-01-07 at 14 13 30

### CODE REVIEW

File: app/views/_includes/forms/outcome/academic-qualifications-undergraduate.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+{% set items = items | push({
+  text: "Other academic award",
+  value: "Other",
+  conditional: {
+    html: academicQualificationAutocomplete
+  }
+}) %}
+
+
```
Comment (ralph-hawkins) [inline app/views/_includes/forms/outcome/academic-qualifications-undergraduate.html]:
```suggestion
{% include "_includes/trainee-name-caption.njk" %}
```

File: app/views/record/outcome/academic-qualifications.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  {% endset %}
+
+  {% if record | isPostgraduate %}
+    {% include "_includes/forms/outcome/academic-qualifications-postgraduate.html" %}
+
+  {% elseif record | isUndergraduate %}
+    {% include "_includes/forms/outcome/academic-qualifications-undergraduate.html" %}
+  {% endif %}
+
+  {# {% include "_includes/forms/outcome/academic-qualifications.html" %} #}
```
Comment (ralph-hawkins) [inline app/views/record/outcome/academic-qualifications.html]: Are you leaving this here for a reason?

**Review by ralph-hawkins — APPROVED** [review]
Looks good, a couple of minor things

File: app/views/record/outcome/academic-qualifications.html (hunk 2 above)
Comment (edwardhorsford) [reply app/views/record/outcome/academic-qualifications.html]: No, it's old code before I split the page between undergrad and postgrad. Removed.

**Review by ralph-hawkins — APPROVED** [review]

**Pull request merged**